wagtail/contrib/frontend_cache/tests.py

```diff
@@ -505,6 +537,150 @@ def test_invalidate_specific_location(self):
         purge_url_from_cache("http://example.com/foo")
         self.assertEqual(PURGED_URLS, ["http://example.com/foo"])
 
+    def assertMessageLogged(self, message, log_record):
+        self.assertIn(f"INFO:wagtail.frontendcache:{message}", log_record.output)
+
+    def test_purge_site_default(self):
```

Comment:
Suggestion: I'd be interested in some `assertNumQueries` checks in here

Reply:
Sure thing

Reply:
Implemented in 6dfc8b4

Reply:
This is actually very skewiff due to the database cache backend being in use, so I'm going to revert.